dockerfile/sdk/native.Dockerfile

```diff
@@ -7,11 +7,6 @@ ARG BUILDARCH
 
 RUN wget -O /lib/libwasmvm_muslc.a https://github.com/CosmWasm/wasmvm/releases/download/v1.0.0/libwasmvm_muslc.$(uname -m).a
 
-RUN git clone https://github.com/tendermint/tendermint; \
```

Comment:
the only use case for packaging this tendermint branch binary is osmosis for state sync, since a workaround is required to set the app version after the state sync snapshot is restored. Rather than bundle this with every single heighliner image, there is a separate tendermint image that can be used for this purpose: `ghcr.io/strangelove-ventures/heighliner/tendermint:remotes-origin-callum-app-version`. This could be done with an initContainer flow for example.